2024-02-17 19:56:01,024 - test_login.py:[19] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 19:56:01,024 - test_login.py:[20] - [INFO] - 测试用例名称：test_login
2024-02-17 19:56:01,025 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 19:56:01,025 - test_login.py:[30] - [INFO] - 实际结果是：登录成功
2024-02-17 19:56:01,025 - test_login.py:[31] - [INFO] - 预期结果是：登录成功
2024-02-17 19:56:01,025 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['12345678922qwe']
2024-02-17 19:56:01,025 - test_login.py:[38] - [INFO] - token是：12345678922qwe
2024-02-17 19:56:01,025 - test_login.py:[40] - [INFO] - 执行登录测试成功
2024-02-17 19:56:01,025 - test_login.py:[41] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 19:56:01,027 - test_login.py:[19] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 19:56:01,027 - test_login.py:[20] - [INFO] - 测试用例名称：test_login
2024-02-17 19:56:01,027 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 19:56:01,027 - test_login.py:[30] - [INFO] - 实际结果是：登录成功
2024-02-17 19:56:01,027 - test_login.py:[31] - [INFO] - 预期结果是：登录失败，密码错误
2024-02-17 19:56:01,027 - test_login.py:[43] - [ERROR] - 执行登录测试失败
2024-02-17 19:56:01,027 - test_login.py:[44] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 19:56:01,052 - test_login.py:[19] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 19:56:01,052 - test_login.py:[20] - [INFO] - 测试用例名称：test_login
2024-02-17 19:56:01,052 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 19:56:01,052 - test_login.py:[30] - [INFO] - 实际结果是：登录成功
2024-02-17 19:56:01,052 - test_login.py:[31] - [INFO] - 预期结果是：登录失败，密码错误
2024-02-17 19:56:01,052 - test_login.py:[43] - [ERROR] - 执行登录测试失败
2024-02-17 19:56:01,052 - test_login.py:[44] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 19:56:01,055 - test_login.py:[19] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 19:56:01,055 - test_login.py:[20] - [INFO] - 测试用例名称：test_login
2024-02-17 19:56:01,055 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 19:56:01,055 - test_login.py:[30] - [INFO] - 实际结果是：登录成功
2024-02-17 19:56:01,055 - test_login.py:[31] - [INFO] - 预期结果是：登录失败，密码错误
2024-02-17 19:56:01,055 - test_login.py:[43] - [ERROR] - 执行登录测试失败
2024-02-17 19:56:01,055 - test_login.py:[44] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 19:56:01,060 - test_login.py:[19] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 19:56:01,060 - test_login.py:[20] - [INFO] - 测试用例名称：test_login
2024-02-17 19:56:01,060 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 19:56:01,060 - test_login.py:[30] - [INFO] - 实际结果是：登录成功
2024-02-17 19:56:01,060 - test_login.py:[31] - [INFO] - 预期结果是：登录失败，请输入账号
2024-02-17 19:56:01,060 - test_login.py:[43] - [ERROR] - 执行登录测试失败
2024-02-17 19:56:01,060 - test_login.py:[44] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 19:56:01,063 - test_login.py:[19] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 19:56:01,063 - test_login.py:[20] - [INFO] - 测试用例名称：test_login
2024-02-17 19:56:01,063 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 19:56:01,063 - test_login.py:[30] - [INFO] - 实际结果是：登录成功
2024-02-17 19:56:01,063 - test_login.py:[31] - [INFO] - 预期结果是：登录失败，请输入账号
2024-02-17 19:56:01,063 - test_login.py:[43] - [ERROR] - 执行登录测试失败
2024-02-17 19:56:01,064 - test_login.py:[44] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 19:56:01,066 - test_login.py:[19] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 19:56:01,066 - test_login.py:[20] - [INFO] - 测试用例名称：test_login
2024-02-17 19:56:01,066 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 19:56:01,066 - test_login.py:[30] - [INFO] - 实际结果是：登录成功
2024-02-17 19:56:01,066 - test_login.py:[31] - [INFO] - 预期结果是：登录失败，请输入账号
2024-02-17 19:56:01,066 - test_login.py:[43] - [ERROR] - 执行登录测试失败
2024-02-17 19:56:01,066 - test_login.py:[44] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 19:56:01,072 - test_login.py:[19] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 19:56:01,072 - test_login.py:[20] - [INFO] - 测试用例名称：test_login
2024-02-17 19:56:01,072 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 19:56:01,072 - test_login.py:[30] - [INFO] - 实际结果是：登录成功
2024-02-17 19:56:01,072 - test_login.py:[31] - [INFO] - 预期结果是：登录失败，请输入密码
2024-02-17 19:56:01,072 - test_login.py:[43] - [ERROR] - 执行登录测试失败
2024-02-17 19:56:01,072 - test_login.py:[44] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 19:56:01,075 - test_login.py:[19] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 19:56:01,075 - test_login.py:[20] - [INFO] - 测试用例名称：test_login
2024-02-17 19:56:01,075 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 19:56:01,075 - test_login.py:[30] - [INFO] - 实际结果是：登录成功
2024-02-17 19:56:01,075 - test_login.py:[31] - [INFO] - 预期结果是：登录失败，请输入密码
2024-02-17 19:56:01,075 - test_login.py:[43] - [ERROR] - 执行登录测试失败
2024-02-17 19:56:01,077 - test_login.py:[44] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 19:56:01,080 - test_login.py:[19] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 19:56:01,080 - test_login.py:[20] - [INFO] - 测试用例名称：test_login
2024-02-17 19:56:01,080 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 19:56:01,080 - test_login.py:[30] - [INFO] - 实际结果是：登录成功
2024-02-17 19:56:01,080 - test_login.py:[31] - [INFO] - 预期结果是：登录失败，请输入密码
2024-02-17 19:56:01,080 - test_login.py:[43] - [ERROR] - 执行登录测试失败
2024-02-17 19:56:01,080 - test_login.py:[44] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:16:26,133 - run.py:[20] - [INFO] - ********************【开始运行测试用例】********************
2024-02-17 20:16:26,133 - run.py:[21] - [INFO] - 运行host地址：https://xxx，设置超时时间：10秒

2024-02-17 20:19:33,940 - run.py:[20] - [INFO] - ********************【开始运行测试用例】********************
2024-02-17 20:19:33,940 - run.py:[21] - [INFO] - 运行host地址：https://xxx，设置超时时间：10秒

2024-02-17 20:19:34,614 - conftest.py:[19] - [INFO] - fixture：请求登录接口，获取token,0.08063422287591793
2024-02-17 20:19:34,614 - test_login.py:[20] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:19:34,614 - test_login.py:[21] - [INFO] - 测试用例名称：test_login
2024-02-17 20:19:34,614 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 20:19:34,614 - test_login.py:[31] - [INFO] - 实际结果是：登录成功
2024-02-17 20:19:34,614 - test_login.py:[32] - [INFO] - 预期结果是：登录成功
2024-02-17 20:19:34,614 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['12345678922qwe']
2024-02-17 20:19:34,614 - test_login.py:[39] - [INFO] - token是：12345678922qwe
2024-02-17 20:19:34,614 - test_login.py:[41] - [INFO] - 执行登录测试成功
2024-02-17 20:19:34,614 - test_login.py:[42] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:19:34,630 - test_login.py:[20] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:19:34,630 - test_login.py:[21] - [INFO] - 测试用例名称：test_login
2024-02-17 20:19:34,630 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 20:19:34,630 - test_login.py:[31] - [INFO] - 实际结果是：登录成功
2024-02-17 20:19:34,630 - test_login.py:[32] - [INFO] - 预期结果是：登录失败，密码错误
2024-02-17 20:19:34,630 - test_login.py:[44] - [ERROR] - 执行登录测试失败
2024-02-17 20:19:34,630 - test_login.py:[45] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:19:34,657 - test_login.py:[20] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:19:34,657 - test_login.py:[21] - [INFO] - 测试用例名称：test_login
2024-02-17 20:19:34,657 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 20:19:34,657 - test_login.py:[31] - [INFO] - 实际结果是：登录成功
2024-02-17 20:19:34,657 - test_login.py:[32] - [INFO] - 预期结果是：登录失败，密码错误
2024-02-17 20:19:34,657 - test_login.py:[44] - [ERROR] - 执行登录测试失败
2024-02-17 20:19:34,658 - test_login.py:[45] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:19:34,663 - test_login.py:[20] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:19:34,663 - test_login.py:[21] - [INFO] - 测试用例名称：test_login
2024-02-17 20:19:34,663 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 20:19:34,663 - test_login.py:[31] - [INFO] - 实际结果是：登录成功
2024-02-17 20:19:34,663 - test_login.py:[32] - [INFO] - 预期结果是：登录失败，密码错误
2024-02-17 20:19:34,663 - test_login.py:[44] - [ERROR] - 执行登录测试失败
2024-02-17 20:19:34,663 - test_login.py:[45] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:19:34,669 - test_login.py:[20] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:19:34,669 - test_login.py:[21] - [INFO] - 测试用例名称：test_login
2024-02-17 20:19:34,669 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 20:19:34,669 - test_login.py:[31] - [INFO] - 实际结果是：登录成功
2024-02-17 20:19:34,669 - test_login.py:[32] - [INFO] - 预期结果是：登录失败，请输入账号
2024-02-17 20:19:34,669 - test_login.py:[44] - [ERROR] - 执行登录测试失败
2024-02-17 20:19:34,669 - test_login.py:[45] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:19:34,674 - test_login.py:[20] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:19:34,674 - test_login.py:[21] - [INFO] - 测试用例名称：test_login
2024-02-17 20:19:34,674 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 20:19:34,674 - test_login.py:[31] - [INFO] - 实际结果是：登录成功
2024-02-17 20:19:34,674 - test_login.py:[32] - [INFO] - 预期结果是：登录失败，请输入账号
2024-02-17 20:19:34,674 - test_login.py:[44] - [ERROR] - 执行登录测试失败
2024-02-17 20:19:34,674 - test_login.py:[45] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:19:34,677 - test_login.py:[20] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:19:34,677 - test_login.py:[21] - [INFO] - 测试用例名称：test_login
2024-02-17 20:19:34,677 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 20:19:34,677 - test_login.py:[31] - [INFO] - 实际结果是：登录成功
2024-02-17 20:19:34,677 - test_login.py:[32] - [INFO] - 预期结果是：登录失败，请输入账号
2024-02-17 20:19:34,677 - test_login.py:[44] - [ERROR] - 执行登录测试失败
2024-02-17 20:19:34,677 - test_login.py:[45] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:19:34,685 - test_login.py:[20] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:19:34,685 - test_login.py:[21] - [INFO] - 测试用例名称：test_login
2024-02-17 20:19:34,685 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 20:19:34,685 - test_login.py:[31] - [INFO] - 实际结果是：登录成功
2024-02-17 20:19:34,685 - test_login.py:[32] - [INFO] - 预期结果是：登录失败，请输入密码
2024-02-17 20:19:34,685 - test_login.py:[44] - [ERROR] - 执行登录测试失败
2024-02-17 20:19:34,685 - test_login.py:[45] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:19:34,691 - test_login.py:[20] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:19:34,691 - test_login.py:[21] - [INFO] - 测试用例名称：test_login
2024-02-17 20:19:34,691 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 20:19:34,691 - test_login.py:[31] - [INFO] - 实际结果是：登录成功
2024-02-17 20:19:34,691 - test_login.py:[32] - [INFO] - 预期结果是：登录失败，请输入密码
2024-02-17 20:19:34,691 - test_login.py:[44] - [ERROR] - 执行登录测试失败
2024-02-17 20:19:34,691 - test_login.py:[45] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:19:34,696 - test_login.py:[20] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:19:34,696 - test_login.py:[21] - [INFO] - 测试用例名称：test_login
2024-02-17 20:19:34,696 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 20:19:34,696 - test_login.py:[31] - [INFO] - 实际结果是：登录成功
2024-02-17 20:19:34,696 - test_login.py:[32] - [INFO] - 预期结果是：登录失败，请输入密码
2024-02-17 20:19:34,696 - test_login.py:[44] - [ERROR] - 执行登录测试失败
2024-02-17 20:19:34,696 - test_login.py:[45] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:19:34,701 - test_login.py:[49] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:19:34,702 - test_login.py:[50] - [INFO] - 测试用例名称：test_register
2024-02-17 20:19:34,702 - test_login.py:[54] - [INFO] - token是：0.08063422287591793
2024-02-17 20:19:34,702 - test_login.py:[58] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:19:59,892 - run.py:[20] - [INFO] - ********************【开始运行测试用例】********************
2024-02-17 20:19:59,892 - run.py:[21] - [INFO] - 运行host地址：https://xxx，设置超时时间：10秒

2024-02-17 20:21:00,684 - run.py:[20] - [INFO] - ********************【开始运行测试用例】********************
2024-02-17 20:21:00,684 - run.py:[21] - [INFO] - 运行host地址：https://xxx，设置超时时间：10秒

2024-02-17 20:21:01,920 - conftest.py:[32] - [INFO] - fixture：请求登录接口，获取token:0.40959380394939005
2024-02-17 20:21:01,920 - conftest.py:[35] - [INFO] - 首次执行，token是：0.40959380394939005
2024-02-17 20:21:01,920 - test_login.py:[20] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:21:01,920 - test_login.py:[21] - [INFO] - 测试用例名称：test_login
2024-02-17 20:21:01,920 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 20:21:01,920 - test_login.py:[31] - [INFO] - 实际结果是：登录成功
2024-02-17 20:21:01,920 - test_login.py:[32] - [INFO] - 预期结果是：登录成功
2024-02-17 20:21:01,920 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['12345678922qwe']
2024-02-17 20:21:01,920 - test_login.py:[39] - [INFO] - token是：12345678922qwe
2024-02-17 20:21:01,920 - test_login.py:[41] - [INFO] - 执行登录测试成功
2024-02-17 20:21:01,920 - test_login.py:[42] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:21:01,920 - test_login.py:[20] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:21:01,920 - test_login.py:[21] - [INFO] - 测试用例名称：test_login
2024-02-17 20:21:01,920 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 20:21:01,920 - test_login.py:[31] - [INFO] - 实际结果是：登录成功
2024-02-17 20:21:01,920 - test_login.py:[32] - [INFO] - 预期结果是：登录失败，密码错误
2024-02-17 20:21:01,930 - test_login.py:[44] - [ERROR] - 执行登录测试失败
2024-02-17 20:21:01,930 - test_login.py:[45] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:21:01,957 - test_login.py:[20] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:21:01,957 - test_login.py:[21] - [INFO] - 测试用例名称：test_login
2024-02-17 20:21:01,957 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 20:21:01,957 - test_login.py:[31] - [INFO] - 实际结果是：登录成功
2024-02-17 20:21:01,957 - test_login.py:[32] - [INFO] - 预期结果是：登录失败，密码错误
2024-02-17 20:21:01,957 - test_login.py:[44] - [ERROR] - 执行登录测试失败
2024-02-17 20:21:01,957 - test_login.py:[45] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:21:01,963 - test_login.py:[20] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:21:01,963 - test_login.py:[21] - [INFO] - 测试用例名称：test_login
2024-02-17 20:21:01,963 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 20:21:01,963 - test_login.py:[31] - [INFO] - 实际结果是：登录成功
2024-02-17 20:21:01,963 - test_login.py:[32] - [INFO] - 预期结果是：登录失败，密码错误
2024-02-17 20:21:01,963 - test_login.py:[44] - [ERROR] - 执行登录测试失败
2024-02-17 20:21:01,963 - test_login.py:[45] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:21:01,978 - test_login.py:[49] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:21:01,978 - test_login.py:[50] - [INFO] - 测试用例名称：test_register
2024-02-17 20:21:01,978 - test_login.py:[54] - [INFO] - token是：0.40959380394939005
2024-02-17 20:21:01,978 - test_login.py:[58] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:21:01,983 - conftest.py:[29] - [INFO] - fixture：读取缓存文件，token是：0.40959380394939005
2024-02-17 20:21:01,983 - test_login.py:[20] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:21:01,983 - test_login.py:[21] - [INFO] - 测试用例名称：test_login
2024-02-17 20:21:01,983 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 20:21:01,983 - test_login.py:[31] - [INFO] - 实际结果是：登录成功
2024-02-17 20:21:01,983 - test_login.py:[32] - [INFO] - 预期结果是：登录失败，请输入账号
2024-02-17 20:21:01,983 - test_login.py:[44] - [ERROR] - 执行登录测试失败
2024-02-17 20:21:01,983 - test_login.py:[45] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:21:02,020 - test_login.py:[20] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:21:02,020 - test_login.py:[21] - [INFO] - 测试用例名称：test_login
2024-02-17 20:21:02,020 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 20:21:02,020 - test_login.py:[31] - [INFO] - 实际结果是：登录成功
2024-02-17 20:21:02,020 - test_login.py:[32] - [INFO] - 预期结果是：登录失败，请输入账号
2024-02-17 20:21:02,020 - test_login.py:[44] - [ERROR] - 执行登录测试失败
2024-02-17 20:21:02,020 - test_login.py:[45] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:21:02,026 - test_login.py:[20] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:21:02,026 - test_login.py:[21] - [INFO] - 测试用例名称：test_login
2024-02-17 20:21:02,031 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 20:21:02,031 - test_login.py:[31] - [INFO] - 实际结果是：登录成功
2024-02-17 20:21:02,031 - test_login.py:[32] - [INFO] - 预期结果是：登录失败，请输入账号
2024-02-17 20:21:02,031 - test_login.py:[44] - [ERROR] - 执行登录测试失败
2024-02-17 20:21:02,031 - test_login.py:[45] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:21:02,031 - test_login.py:[20] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:21:02,031 - test_login.py:[21] - [INFO] - 测试用例名称：test_login
2024-02-17 20:21:02,031 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 20:21:02,031 - test_login.py:[31] - [INFO] - 实际结果是：登录成功
2024-02-17 20:21:02,031 - test_login.py:[32] - [INFO] - 预期结果是：登录失败，请输入密码
2024-02-17 20:21:02,031 - test_login.py:[44] - [ERROR] - 执行登录测试失败
2024-02-17 20:21:02,031 - test_login.py:[45] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:21:02,042 - test_login.py:[20] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:21:02,042 - test_login.py:[21] - [INFO] - 测试用例名称：test_login
2024-02-17 20:21:02,042 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 20:21:02,042 - test_login.py:[31] - [INFO] - 实际结果是：登录成功
2024-02-17 20:21:02,042 - test_login.py:[32] - [INFO] - 预期结果是：登录失败，请输入密码
2024-02-17 20:21:02,042 - test_login.py:[44] - [ERROR] - 执行登录测试失败
2024-02-17 20:21:02,042 - test_login.py:[45] - [INFO] - ---------------接口自动化结束---------------

2024-02-17 20:21:02,051 - test_login.py:[20] - [INFO] - ---------------接口自动化开始---------------
2024-02-17 20:21:02,051 - test_login.py:[21] - [INFO] - 测试用例名称：test_login
2024-02-17 20:21:02,051 - json_res.py:[16] - [INFO] - jsonpath匹配结果是: ['登录成功']
2024-02-17 20:21:02,051 - test_login.py:[31] - [INFO] - 实际结果是：登录成功
2024-02-17 20:21:02,051 - test_login.py:[32] - [INFO] - 预期结果是：登录失败，请输入密码
2024-02-17 20:21:02,051 - test_login.py:[44] - [ERROR] - 执行登录测试失败
2024-02-17 20:21:02,051 - test_login.py:[45] - [INFO] - ---------------接口自动化结束---------------

